core/execution/market.go

```diff
@@ -1911,15 +1911,23 @@ func (m *Market) resolveClosedOutParties(ctx context.Context, distressedMarginEv
 		// which have acceptable margins
 		okPos, closed = m.risk.ExpectMargins(distressedMarginEvts, m.lastTradedPrice.Clone())
 
+		parties := make([]string, 0, len(okPos))
+		for _, v := range okPos {
+			parties = append(parties, v.Party())
+		}
 		if m.log.GetLevel() == logging.DebugLevel {
-			for _, v := range okPos {
+			for _, pID := range parties {
 				if m.log.GetLevel() == logging.DebugLevel {
```

Comment:
this isn't needed

Reply:
That's what was already there. I only changed this because I added the loop outside the `if` creating a slice of party ID's, and used that slice here to not repeatedly call `v.Party()`

Reply:
Removed the inner log-level check, and replaced the outer one with `IsDebug()`